lib/view/authentication/login_screen.dart

```diff
@@ -0,0 +1,253 @@
+                              }
+                            },
+                          ),
+                          SizedBox(height: 10),
+                          OrLine(),
+                          SizedBox(height: 25),
+
+                          // Google and github auth widget
+                          AuthIcons(),
+                        ],
+                      ),
+                    ),
+                  ),
+                  SizedBox(
+                    height: 25,
+                  ),
+                  GestureDetector(
+                    onTap: () {
+                      Get.toNamed(RoutesNames.signupScreen);
+                    },
+                    child: Container(
+                      alignment: Alignment.center,
+                      width: width,
+                      padding: EdgeInsets.symmetric(
+                        horizontal: 25,
+                        vertical: 20,
+                      ),
+                      decoration: BoxDecoration(
+                        borderRadius: BorderRadius.circular(8),
+                        color: Color(0xffffffff).withOpacity(0.05),
+                      ),
+                      child: RichText(
+                        text: TextSpan(
+                          children: <TextSpan>[
+                            TextSpan(
+                              text: "Don't Have an account?, ",
+                              style: TextStyle(
+                                fontWeight: FontWeight.w500,
+                              ),
+                            ),
+                            TextSpan(
+                              text: ' Sign Up',
+                              style: TextStyle(
+                                color: Color(0xff8DEBFF),
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                  ),
+                  SizedBox(
+                    height: 20,
+                  ),
+                ],
+              ),
+              Footer(),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class AuthIcons extends StatelessWidget {
+  const AuthIcons({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        GestureDetector(
+          onTap: () {
+            // Google Sign In method
+          },
+          child: SvgPicture.asset(
+            'assets/svg/google.svg',
+            width: 40,
+          ),
+        ),
+        SizedBox(
+          width: 30,
+        ),
+        GestureDetector(
+          onTap: () {
+            // Github auth
+          },
+          child: Image.asset(
+            "assets/images/github.png",
+            width: 40,
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class OrLine extends StatelessWidget {
```

Comment:
Move this to separate file under widgets folder

Reply:
exported to widgets/login_widgets.dart